### ISSUE TITLE
Why do not include the new eROSITA mission?

### ISSUE THREAD

**Issue by alessandropeca** [XRStools/soxs#6]
Why do not include the new eROSITA mission?
I know that it is possible to create new instruments inside SOXS, but including a particle background is not easy.

**Comment by jzuhone** [comment]
Hi @alessandropeca, 

SIXTE already does a marvelous job of simulating eROSITA:

https://www.sternwarte.uni-erlangen.de/research/sixte/index.php

And SIMPUT files which are produced with SOXS / pyXSIM can be used in SIXTE. 

It is admittedly not Python, but at the moment there is not a huge enough drive to replicate what SIXTE does.

**Comment by alessandropeca** [comment]
Thanks for the reply. 
I'm trying to create the eROSITA instrument on SOXS and I have issues building the instrumental bkg. 
I'm using the file provided by SIXTE. Unfortunately, it seems that SIXTE and SOXS use different units for the particle rates.
SOXS seems to easy to use and, as you said, is written in python, that's why I am trying to simulate eROSITA with it.

**Comment by jzuhone** [comment]
@alessandropeca did you have something going yet? 

What we'd need to do is set up all 7 telescopes, which have a different RMF and ARF. Then you'd write 7 event files. 

This is more or less what SIXTE does.

**Comment by alessandropeca** [comment]
Yes, I used the instrument background file provided by SIXTE. I fixed some differences for the SOXS requirements and it seems to work now.
